pkg/iosrc/uri_unix.go

```diff
@@ -0,0 +1,23 @@
+// +build aix darwin dragonfly freebsd js,wasm linux netbsd openbsd solaris
```

Comment:
```suggestion
// +build !windows
```

Reply:
And maybe change the file name to uri_notwindows.go.

Reply:
Yep, thanks